packages/components/src/PaymentAccordion/Component.purs

```diff
@@ -0,0 +1,170 @@
+module KSF.PaymentAccordion.Component where
+
+import Prelude
+
+import Data.Array (length, reverse, zip)
+import Data.Enum (enumFromTo)
+import Data.List as List
+import Data.Date (Date)
+import Data.DateTime (DateTime(..))
+import Data.Formatter.DateTime (FormatterCommand(..))
+import Data.Formatter.DateTime as FormatterD
+import Data.Formatter.Number as FormatterN
+import Data.Maybe (Maybe(..))
+import Data.Tuple (Tuple(..))
+import Effect (Effect)
+import KSF.User (SubscriptionPayments, PaymentState(..), PaymentType(..))
+import React.Basic (JSX, make)
+import React.Basic as React
+import React.Basic.DOM as DOM
+import React.Basic.Events (handler_)
+
+type Self = React.Self Props State
+
+type Props =
+  { payments :: Array SubscriptionPayments
+  }
+
+type State =
+  { focus :: Maybe Int
+  }
+
+jsComponent :: React.Component Props
+jsComponent = component
+
+component :: React.Component Props
+component = React.createComponent "PaymentList"
+
+payments :: Props -> JSX
+payments = make component
+  { initialState: { focus: Nothing }
+  , render
+  , didMount
+  }
+
+didMount :: Self -> Effect Unit
+didMount self = do
+  pure unit
+
+render :: Self -> JSX
+render self =
+  DOM.div
+    { className: "mitt-konto--payments"
```

Comment:
It might be a good idea to separate the css of this particular component into its own file, as it's now dependent on mitt konto. The convention is just to create a `<ComponentName>.less` under the `/less` directory.

Reply:
Done.